pkg/api/v20220904/clustermanager.go

```diff
@@ -0,0 +1,198 @@
+package v20220904
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+// OCM Kinds supported
+const (
+	MachinePoolKind          = "MachinePool"
+	SyncIdentityProviderKind = "SyncIdentityProvider"
+	SyncSetKind              = "SyncSet"
+	SecretKind               = "Secret"
+)
+
+type ClusterManagerConfigurationList struct {
+	ClusterManagerConfigurations []*ClusterManagerConfiguration `json:"value"`
+
+	NextLink string `json:"nextLink,omitempty"`
+}
+
+type ClusterManagerConfiguration struct {
+	// ID is the unique identifier for the cluster manager configuration
+	ID string `json:"id,omitempty"`
+
+	// The resource name
+	Name string `json:"name,omitempty"`
+
+	// The parent cluster resourceId
+	ClusterResourceId string `json:"clusterResourceId,omitempty"`
```

Comment:
```suggestion
	ClusterResourceID string `json:"clusterResourceId,omitempty"`
```

Reply:
any reason not to use `ID` like we do elsewhere? I saw it was changed here
https://github.com/Azure/ARO-RP/pull/2332/files#diff-96fa795b77897d2fd960da3db0061471bf432707afadce4d4eda27159df8ec16R26